microsoft-edge/webview2/get-started/winforms.md

```diff
@@ -68,71 +82,98 @@ Use NuGet to add the WebView2 SDK to the project.
+    :::image type="complex" source="./media/winforms-properties.png" alt-text="Properties of the WebView2 control" lightbox="./media/winforms-properties.png":::
        Properties of the WebView2 control
     :::image-end:::
     
-1.  The `Source` property sets the initial URI displayed in the WebView2 control.  Set the `Source` property to `https://www.microsoft.com`.  
-    
-    :::image type="complex" source="./media/winforms-source.png" alt-text="The Source property of the WebView2 control":::
-       The **Source** property of the WebView2 control
-    :::image-end:::  
+1.  The **Source** property sets the initial URI displayed in the WebView2 control. Set the **Source** property to `https://www.microsoft.com`.
 
-To build and run your project, select `F5`.  Ensure your WebView2 control displays [https://www.microsoft.com][MicrosoftMain].
+1. Select **F5** to build and run your project.
 
-:::image type="complex" source="./media/winforms-hello-webview.png" alt-text="hello webview" lightbox="./media/winforms-hello-webview.png":::
-   hello webview  
-:::image-end:::    
+    Make sure the WebView2 control displays [https://www.microsoft.com][MicrosoftMain].
 
-> [!NOTE]
-> If you are working on a high DPI monitor, you may have to [configure your Windows Forms app for high DPI support][DotnetFrameworkWinformsHighDpiSupportWindowsFormsConfiguringYourWindowsFormsAppForHighDpiSupport].  
+    :::image type="complex" source="./media/winforms-hello-webview.png" alt-text="hello webview" lightbox="./media/winforms-hello-webview.png":::
+       hello webview  
+    :::image-end:::
 
-## Step 4 - Handle Window Resize Events  
+    > [!NOTE]
+    > If you are working on a high DPI monitor, you may have to [configure your Windows Forms app for high DPI support][DotnetFrameworkWinformsHighDpiSupportWindowsFormsConfiguringYourWindowsFormsAppForHighDpiSupport].  
+    
+## Step 4 - Add controls and process window resize events  
 
-Add a few more controls to your Windows Forms from the toolbox, and then handle window resize events appropriately.  
+Add more controls to your Windows Forms from the toolbox, then process window resize events.  
 
-1.  In the **Windows Forms Designer**, open the **Toolbox**.  
-1.  Drag and Drop a **TextBox** into the Windows Forms App.  Name the **TextBox** `addressBar` in the **Properties Tab**.  
-1.  Drag and Drop a **Button** into the Windows Forms App.  Change the text in the **Button** to `Go!` and name the **Button** `goButton` in the **Properties Tab**.  
-    
-    The app should look like the following image in the designer.  
+1. Choose **View** > **Toolbox**.
+1. In the **Toolbox** choose **Common Controls**. 
+1. Drag and drop **TextBox** control into the Windows Forms App.
+1. In the Properties panel, change the **\(Name\)** to **addressBar**.  
+1. Drag and Drop a **Button** control into the Windows Forms App.
```

Comment:
"Drag and drop", or just "Drag foo into bar."  Check https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui